### ISSUE TITLE
support Overture Maps

### ISSUE THREAD

**Issue by ambuehll** [eclipse-sumo/sumo#14981]
Hello,

I wanted to raise the suggestion to have a look at Overture Maps (https://overturemaps.org), which might bring an enhancement to OSM. 
Just as an alternative to OSM imports. 
We'll be looking into this in more details and potentially report here.

**Comment by namdre** [comment]
Data can be downloaded with https://github.com/OvertureMaps/overturemaps-py/
However, the data I retrieved all came with the info `dataset":"OpenStreetMap"`.

**Comment by ambuehll** [comment]
yes, I think currently it's only OSM, but supposedly at some point it should be enriched. I had the impression it will be a slightly altered format, but we'll have a look at it. cc: @gladysmcgan
